Doc/library/fractions.rst

```diff
@@ -125,8 +125,7 @@ another rational number, or from a string.
 
    .. attribute:: denominator
 
-      Denominator of the Fraction in lowest term.
-
+      Denominator of the Fraction in lowest term. It is always positive.
```

Comment:
```suggestion
      Denominator of the Fraction in lowest terms and positive.
```

numerator description also should look like "Numerator of the Fraction in lowest terms.", as e.g. help(int.numerator).